packages/types/farFU.cfc

```diff
@@ -806,10 +806,9 @@
 				<cfif not isvalid("integer",stResult.__redirectionType)>
```

Comment:
On the line before this there is a <cftry> without any matching closing </cftry>, perhaps it should also be removed?